builders/build/runtime-upgrades.md

```diff
@@ -17,60 +17,61 @@ The following table contains a list of the runtime upgrades and the block at whi
+| [2201](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2201) | [3290853](https://moonbeam.subscan.io/block/3290853){target=_blank} | [3858885](https://moonriver.subscan.io/block/3858885){target=_blank} | [3842850](https://moonbase.subscan.io/block/3842850){target=_blank} |
+| [2301](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2301) |                                  -                                  |                                  -                                   | [4172407](https://moonbase.subscan.io/block/4172407){target=_blank} |
+| [2302](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2302) | [3456477](https://moonbeam.subscan.io/block/3456477){target=_blank} | [4133065](https://moonriver.subscan.io/block/4133065){target=_blank} | [4193323](https://moonbase.subscan.io/block/4193323){target=_blank} |
+| [2401](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2401) |                                  -                                  | [4668844](https://moonriver.subscan.io/block/4668844){target=_blank} | [4591616](https://moonbase.subscan.io/block/4591616){target=_blank} |
+| [2402](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2402) |                                  -                                  |                                  -                                   | [4772817](https://moonbase.subscan.io/block/4772817){target=_blank} |
+| [2403](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2403) | [4163078](https://moonbeam.subscan.io/block/4163078){target=_blank} | [4770488](https://moonriver.subscan.io/block/4770488){target=_blank} | [4804425](https://moonbase.subscan.io/block/4804425){target=_blank} |
+| [2500](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2500) |                                  -                                  | [5175574](https://moonriver.subscan.io/block/5175574){target=_blank} | [5053547](https://moonbase.subscan.io/block/5053547){target=_blank} |
+| [2501](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2501) | [4543267](https://moonbeam.subscan.io/block/4543267){target=_blank} | [5211264](https://moonriver.subscan.io/block/5211264){target=_blank} | [5194594](https://moonbase.subscan.io/block/5194594){target=_blank} |
+| [2601](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2601) |                                  -                                  |                                  -                                   | [5474345](https://moonbase.subscan.io/block/5474345){target=_blank} |
+| [2602](https://github.com/moonbeam-foundation/moonbeam/releases/tag/runtime-2602) | [4977160](https://moonbeam.subscan.io/block/4977160){target=_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=_blank} |
```

Comment:
the github link returns a 404

Reply:
True! We've not released notes for these two yet